p/themes/base-theme/frss.css

```diff
@@ -48,8 +48,7 @@
 	font-family: 'OpenSans';
 	font-style: normal;
 	font-weight: 400;
-	src: local('Open Sans'), local('OpenSans'),
```

Comment:
It might be worth pointing out that I do in fact have a local Open Sans (possibly not on Windows).